src/routes/userRoutes.js

```diff
@@ -81,6 +81,31 @@ routes.deleteUser = async (req, res) => {
       res.status(500).json({error: {message: 'Deleting user failed. Please try again'}})
   }
 }
+  routes.updateDBPassword = async(req,res) =>{
+    if(req.params.id===null ||!req.body.password ){
```

Comment:
I wonder if `req.params.id === null` is correct. If not passing any id parameter set `req.params.id` value as `null`, it makes sense. But what if it's `undefined` or empty string? We can do `!req.params.id` instead

Reply:
So you think writing !req.params.id is better choice? In which case do you think it could be undefined?

Reply:
I'd rather do !req.params.id because I think when frontend send request with no id parameter value, it will be undefined. I didn't tested so I don't know the answer. But !req.params.id works for both null and undefined so I'd rather do this way